scripts/install-worker.sh

```diff
@@ -73,8 +72,20 @@ sudo yum install -y \
   mdadm \
   pigz
 
-# Remove any old kernel versions. `--count=1` here means "only leave 1 kernel version installed"
-sudo package-cleanup --oldkernels --count=1 -y
+# packages that need special handling
+SPECIAL_PACKAGES=(
+  "curl"            # exists in both AL2 and AL2023 (need to overwrite curl-minimal)
```

Comment:
hopefully you don't actually *need* some of the functionality not in `curl-minimal`. If you do though, we want to know :)

Reply:
ack, will switch back to just curl-minimal